src/game.cpp

```diff
@@ -5823,6 +5823,13 @@ void game::print_terrain_info( const tripoint &lp, const catacurses::window &w_l
     // Cover percentage from terrain and furniture next.
     fold_and_print( w_look, point( column, ++line ), max_width, c_light_gray, _( "Cover: %d%%" ),
                     m.coverage( lp ) );
+
+    if( m.has_flag( ter_furn_flag::TFLAG_TREE, lp ) ) {
+        const int lines = fold_and_print( w_look, point( column, ++line ), max_width, c_light_gray,
+                                          _( "Can be %s with the right tools." ), colorize( _( "cut down" ), c_green ) );
```

Comment:
```suggestion
                                          _( "Can be <color_green>cut down</color> with the right tools." ) );
```

I don't insist on my variant, but it's just one less string to translate.

Reply:
I tried that (or something very similar) initially and it didn't convert the tags to the color, which is why I switched it.

Reply:
It appears to be working now, so I must have screwed something up before. Good to know this method works for future use!